### DISCUSSION TITLE
2023/02/remote-stb-openwrt-dengan-cloudflared

### DISCUSSION THREAD

**Discussion by giscus** [madi10/MANTANKODE#34] [General]
# 2023/02/remote-stb-openwrt-dengan-cloudflared

Tutorial Remote OpenWrt dari jarak jauh dengan menggunakan Tunnel Cloudflare.

https://www.mantankode.net/2023/02/remote-stb-openwrt-dengan-cloudflared.html?m=1

**Comment by madi10** [comment]
Pertama dong, hehe
